### DISCUSSION TITLE
During the data preprocessing stage, specifically when segmenting clothes rather than generating grasping poses, is it possible to adjust the segmentation model parameters for different types of clothing?

### DISCUSSION THREAD

**Discussion by Victorlouisdg** [Victorlouisdg/cloth-competition#23] [General]
**Question:** During the data preprocessing stage, specifically when segmenting clothes rather than generating grasping poses, is it possible to adjust the segmentation model parameters for different types of clothing?

**Answer:** No, the entire grasp selection process must be autonomous, including segmentation. This means your segmentation method should be adaptable enough to handle variations in clothing types without manual parameter adjustments. We impose this constraint because, in real-world scenarios, you also wouldn't always know the exact type of garment in advance (especially because before unfolding is completed, it can be harder to classify garments).

**Comment by GostInShell** [comment]
@Victorlouisdg What about categories, i.e., short-sleeved shirts and towels? Since classfication may not be the focus of this competition, does our code allow to know whether we are grasping shirts or towels in advance?

**Reply by Victorlouisdg** [reply]
Great question! You're absolutely right that this competition focuses on the unfolding process itself, rather than classifying clothing. Classification can be an optional step, potentially introducing an extra point of failure for some methods, while others work regardless of the cloth category (category-agnostic methods).

If your chosen method requires knowing the category, you'll need to incorporate a way to classify the cloth autonomously within your code. To keep the competition fair, **manual input of the cloth category isn't allowed**, as it would provide additional information and create an unfair advantage for certain methods.
